smg-automotive/components-pkg
# feat: add optimizer icon

References [VSST-2504](https://smg-au.atlassian.net/browse/VSST-2504)

## Motivation and context

Missing icon for optimizer. It exists in design system and on Frontify. 

## Before

No Optimizer icon in components library

## After
<img width="194" alt="Screenshot 2024-07-24 at 15 56 39" src="https://github.com/user-attachments/assets/85103053-cf51-4af2-9e6f-6a32a5c9e3d9">


## How to test

Check Optimizer icon
https://feat-add-optimizer-icon-components-pkg.branch.autoscout24.dev/?path=/docs/foundations-icons--documentation
